install.sh

```diff
@@ -85,13 +90,59 @@ pacman --noconfirm -S \
 	grub \
 	steam
 
-# install nvidia graphics if needed
-if lspci | grep -E -i '(vga|3d)' | grep -i nvidia > /dev/null; then
-	pacman --noconfirm -S nvidia nvidia-utils lib32-nvidia-utils
-fi
-
 systemctl enable NetworkManager lightdm bluetooth
 
+# gpu detection/driver installation
+elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon, amdgpu' > /dev/null; then
```

Comment:
will this ever be true when installing from the Arch linux installer which is command line only?

Reply:
@tokyoraven02

Reply:
We have to test this out. The thing I said earlier about kernel modules still applies here.